OpenRA.Game/TextNotification.cs

```diff
@@ -21,16 +21,18 @@ public class TextNotification : IEquatable<TextNotification>
 		public readonly TextNotificationPool Pool;
 		public readonly string Prefix;
 		public readonly string Text;
-		public readonly Color PrefixColor;
-		public readonly Color TextColor;
+		public readonly Color? PrefixColor;
+		public readonly Color? TextColor;
+		public readonly DateTime Time;
 
-		public TextNotification(TextNotificationPool pool, string prefix, string text, Color prefixColor, Color textColor)
+		public TextNotification(TextNotificationPool pool, string prefix, string text, Color? prefixColor, Color? textColor)
 		{
 			Pool = pool;
 			Prefix = prefix;
 			Text = text;
 			PrefixColor = prefixColor;
 			TextColor = textColor;
+			Time = DateTime.Now;
```

Comment:
I imagine this will break if we ever want to expose timestamps for replays or savegames, but there are bigger issues blocking that than this. Not asking you to change anything, I just thought it important to raise this.

Reply:
I didn't consider this when adding the timestamps to the in-game menu chat. This is only necessary for the lobby, so I'll remove the timestamps from the in-game chat.

Reply:
Its fine for the ingame menu chat too, thats not visible in replays or singleplayer (and we dont support multiplayer saves). IMO we should keep it.